### PR TITLE
Update documentation for local running

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: docs/run-local.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ You may wish to run the dashboard. Instructions are [here](run-dashboard.md).
 
 You may wish to run Rancher rather than the dashboard. Rancher provide a step by step walkthrough to install, which is linked [here](rancher-setup.md)
 
+## Add Dependencies
+
+In order to run locally, a dependancy to the ACR must be added to Helm. This can be done using the below command
+```
+helm repo add gpitfuturesdevacr https://gpitfuturesdevacr.azurecr.io/helm/v1/repo --username gpitfuturesdevacr --password $(gpitfuturesdevacr-pass)
```
Comment (sam-roberts-bjss) [inline docs/run-local.md:47]: How do I know where to get `$(gpitfuturesdevacr-pass)` information?

File: environments/local-namespace.yml [hunk 2]
```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    app: buyingcatalogue-dev
+    app: buyingcatalogue
```
Comment (sam-roberts-bjss) [inline environments/local-namespace.yml:5]: Are you sure this label should have changed?

File: docs/run-local.md (hunk 1 above)
Comment (PhysicsJon) [reply docs/run-local.md:47]: You would already have it if starting from scratch (it's needed for the Private registry a couple of steps above)

File: environments/local-namespace.yml (hunk 2 above)
Comment (stuart-bass-bjss) [reply environments/local-namespace.yml:5]: yes, this should be buyingcatalogue. It's a mistake on my part to add the -dev

File: docs/run-local.md (hunk 1 above)
Comment (stuart-bass-bjss) [reply docs/run-local.md:47]: We should probably add clearer instructions that it's the same password as used for the local docker command

Comment (sam-roberts-bjss) [reply docs/run-local.md:47]: In that case doesn't this section belong in the Private registry section?

File: environments/local-namespace.yml (hunk 2 above)
Comment (PhysicsJon) [reply environments/local-namespace.yml:5]: Had confirmation that -dev was to be removed

**Pull request merged**